tests/test_scrubbers.py

```diff
@@ -29,3 +30,16 @@ def test_hash_scrubber_textfield(self):
         data.refresh_from_db()
 
         self.assertNotEqual(data.description, 'Foo')
+
+    def test_faker_scrubber_date_in_past(self):
```

Comment:
nitpicking: you are actually not interested in the `date_in_past` part. You're just using that one specific provider as an example, but the test is actually about passing arbitrary arguments to faker providers.
```suggestion
    def test_faker_scrubber_with_provider_arguments(self):
```